app/forms/admin/create_topup_form.rb

```diff
@@ -0,0 +1,60 @@
+module Admin
+  class CreateTopupForm
+    include ActiveModel::Model
+    include ActiveModel::Attributes
+    include ActionView::Helpers::NumberHelper
+
+    def self.model_name
+      ActiveModel::Name.new(self, nil, "Topup")
+    end
+
+    attr_reader :claim, :created_by
+
+    attribute :award_amount, :decimal
+
+    validate :topup_is_valid
+    validate :note_is_valid
+
+    def initialize(claim:, created_by:, params: {})
+      @claim = claim
+      @created_by = created_by
+
+      super(params)
+    end
```

Comment:
pretty sure you don't need to implement `initialize` and will work if you just delete this method?